scss/modules/_nav.scss

```diff
@@ -181,14 +183,12 @@
 .site-nav__subtitle {
   display: block;
   margin: 0;
+  font-size: 1.4rem;
   padding: 1rem;
-
-  @include media($lg) {
-    border-bottom: 1px solid $beige;
-  }
 }
 
 .site-nav__dropdown {
+  font-family: $sans-serif;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered display, font-family, padding-bottom